css/message-center.css

```diff
@@ -0,0 +1,109 @@
+main {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.page-title {
+    font-size: 30px;
+    margin-bottom: 10px;
+}
+
+.main__wrapper {
+    margin-top: 40px;
+    padding: 10px 30px;
+    width: 80%;
+    max-width: 800px;
+}
+
+.msg-container {
+    background-color: #f2f2f2;
+    border-radius: 20px;
+    min-height: 60vh;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 20px;
+    align-items: center;
+    overflow: hidden;
+}
+
+.msg-row:nth-child(even){
+    background-color: var(--color-grey);
+}
+
+
+.msg-nav {
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+}
+
+.msg-nav-item {
```

Comment:
Can you add a `cursor: pointer;` to this class so it's more clear that inbox and sent should be clickable? They don't have to work for now but I find it a unintuitive if the cursor doesn't change.